File: 백준/Bronze/2562. 최댓값/최댓값.py
num_lst = []

p = 0
b = 0

for i in range(9):
    a = int(input())
    num_lst.append(a)

for j in range(9):
    if num_lst[j] > p:
        p = num_lst[j]
        b = j+1

print(p)
print(b)

File: 백준/Bronze/2562. 최댓값/최댓값.js
const fs = require("fs");
const filePath = process.platform === "linux" ? "/dev/stdin" : "./input.txt";
const input = fs
  .readFileSync(filePath)
  .toString()
  .trim()
  .split("\n")
  .map(Number);

let max_num = 0;

max_num = Math.max(...input);
console.log(max_num);

console.log(input.indexOf(max_num) + 1);
